packages/shared/src/styles/base.css

```diff
@@ -31,6 +31,7 @@ body {
 
   overflow-x: hidden;
 
+  @apply antialiased;
```

Comment:
Saw some pages doesn't have a wrapper with `.antialiased` class, so added it to global CSS so it's here instead. Then we can later remove the wrappers instead